request.h

```diff
@@ -0,0 +1,20 @@
+#ifndef REQUEST_H
+#define REQUEST_H
+
+#include <string>
+
+class Request
+{
+    private:
```

Comment:
In most cases it makes sense to put private section below public - public is more important.